### PR TITLE
fix(#593): Run Browser tests in parallel & Upload assets to github with release

### LINKED ISSUE #593: Build static assets in CI
### What needs to be done
 
Build the production static assets in CI and add them as attachments to the github release

CSU Prod doesn't have enough CPU/Memory allocated to build the assets in production.

### PR DESCRIPTION
**Pull request by alexstojda**
##### _List all issues this PR resolves_
- closes #593

##### _Complete these items before requesting reviews_
- [ ] Static Analysis Rules Validated
- [ ] All commits follow naming conventions

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -107,13 +132,13 @@ jobs:
           command: php artisan migrate:fresh --seed
       - run:
           name: "Run Tests"
-          command: php artisan test
+          command: php artisan test --filter Browser
```
Comment (EvanDime) [inline .circleci/config.yml:135]: Rename to `Run Browser Tests` ?

**Review by EvanDime — DISMISSED** [review]
one comment but it is technically fine if to leave it

File: .circleci/config.yml [hunk 2]
```diff
@@ -106,14 +129,14 @@ jobs:
...
-          command: php artisan test
+          name: "Run Browser Tests"
+          command: php artisan test --filter Browser
       - store_test_results:
           path: reports
       - store_artifacts:
           path: reports
       - store_artifacts:
-          path: tests/Browser/screnshots
+          path: tests/Browser/screenshots
```
Comment (a-drew) [inline .circleci/config.yml:139]: very important typo fix 🚀

**Review by a-drew — APPROVED** [review]
awesome job exporting the release step into a script we can reuse 🥇

**Review by EvanDime — APPROVED** [review]

**Pull request merged**